packages/cw-denom/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "cw-denom"
+version = "0.1.0"
+edition = "2021"
+authors = ["ekez ekez@withoutdoing.com"]
+repository = "https://github.com/DA0-DA0/dao-contracts"
+description = "A package for validation and handling of cw20 and native Cosmos SDK denominations."
+
+
+[dependencies]
+cosmwasm-std = "1.0"
+schemars = "0.8"
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+thiserror = { version = "1.0" }
+cw20 = "0.13"
```

Comment:
why not using the latest (v.015.1)?

Reply:
we're not yet on cosmwasm 1.1. will update soon as a followup to this pr!